src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -206,6 +206,17 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                 }
             },
 
+            innerText: {
+                condition: el => typeof el.tagName === 'string' && el.tagName.toLowerCase() === 'script',
```

Comment:
`innerText` is not in spec and available only in IE and Chrome. We need to add check in the condition for the presence of the `innerText` property